Unable to use System.Text.Json iin Jupyter c# kernel

#### Describe the bug
I just installed the c# jupyter kernel today for trying out. I can not get System.Text.Json namespace to import. If I do 
```using System.Text.Json;``` 
I get this error
 ```(1,19): error CS0234: The type or namespace name 'Json' does not exist in the namespace 'System.Text' (are you missing an assembly reference?)```
I also tried to reference it as a nuget package, with 
```#r "nuget:System.Text.Json"``` 
and it complete successful, but I still cannot use the namespace.

If I run ```Environment.Version``` from within the notebook I get 3.0.0 back. And ```dotnet --version``` shows 3.0.100 so I think I have the proper version installed.

I have installed some other packages, for example MongoDB.Driver could be installed with nuget and imported with using just fine, so it seems its something specifically with this package.

#### Did this error occur while using `dotnet try` or online?
- [x] `dotnet-try`
- [ ] online

#### What kind of error was it?
- [ ] User Interface (UI):  For example the output never displayed 
- [ ] Service Error: For example "The service is temporarily unavailable. We are working on it"
- [x] Other:

#### Screenshots
![image](https://user-images.githubusercontent.com/1195272/68773618-32dea080-062c-11ea-864a-45b42dd6b7a5.png)

#### Please complete the following:
 - OS
    - [x] Windows 10
    - [ ] macOS
    - [ ] Linux (Please specify distro)
    - [ ] iOS
    - [ ] Android
 - Browser 
    - [ ] Chrome
    - [ ] Edge
    - [ ] Safari
    - [x] Firefox

--- a/Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs
+++ b/Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs
@@ -93,6 +93,44 @@ namespace Microsoft.DotNet.Interactive.Tests
                 .Be(5);
         }
 
+        // Option 1: inline switch
+        [Theory]
+        [InlineData(Language.CSharp)]
+        //[InlineData(Language.FSharp)]    Todo: Issue 695 - dotnet-interactive with an F# notebook does not load System.Text.Json #695
+        public async Task it_can_reference_system_text_json(Language language)
+        {
+            var source = language switch
+            {
+                Language.FSharp => new[]
+                {
+@"open System.Text.Json;
+let jsonException = JsonException()
+let location = jsonException.GetType().Assembly.Location
+location.EndsWith(""System.Text.Json.dll"")"
+                },
+
+                Language.CSharp => new[]
+                {
+@"using System.Text.Json;
+var jsonException = new JsonException();
+var location = jsonException.GetType().Assembly.Location;
+location.EndsWith(""System.Text.Json.dll"")"
+        }
+            };
+
+            var kernel = CreateKernel(language);
+
+            await SubmitCode(kernel, source);
+
+            KernelEvents
+                .OfType<ReturnValueProduced>()
+                .Last()
+                .Value
+                .Should()
+                .Be(true);
+        }
+
+
         [Theory]
         [InlineData(Language.FSharp)]
         public async Task kernel_base_ignores_command_line_directives(Language language)
@@ -1564,7 +1602,6 @@ using Microsoft.ML.AutoML;
 
             await kernel.SubmitCodeAsync(@"
 #r ""nuget:System.Text.Json""
-//using System.Text.Json;
 ");
             // It should work, no errors and the requested package should be added
             events.Should()

--- a/Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs
+++ b/Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs
@@ -123,11 +123,12 @@ location.EndsWith(""System.Text.Json.dll"")"
             await SubmitCode(kernel, source);
 
             KernelEvents
-                .OfType<ReturnValueProduced>()
-                .Last()
-                .Value
-                .Should()
-                .Be(true);
+              .Should()
+              .ContainSingle<ReturnValueProduced>()
+              .Which
+              .Value
+              .Should()
+              .Be(true);
         }
 
 

